src/pages/Organization/OrganizationFacilities.tsx

```diff
@@ -114,62 +115,65 @@ export default function OrganizationFacilities({
-                  <div className="relative h-48 bg-gray-100">
-                    {facility.read_cover_image_url ? (
-                      <img
-                        src={facility.read_cover_image_url}
-                        alt={facility.name}
-                        className="w-full h-full object-cover"
-                      />
-                    ) : (
-                      <div className="w-full h-full flex items-center justify-center overflow-hidden">
-                        <Avatar name={facility.name} />
-                      </div>
-                    )}
-                  </div>
-                  <CardContent className="p-6">
-                    <div className="flex flex-col h-full">
-                      <div className="flex items-start justify-between">
-                        <div>
-                          <h3 className="text-md font-medium text-gray-900">
-                            {facility.name}
-                          </h3>
-                          <div className="font-medium">
-                            {facility.facility_type}
-                          </div>
+                <div className="relative h-48 bg-gray-100">
+                  {facility.read_cover_image_url ? (
+                    <img
+                      src={facility.read_cover_image_url}
+                      alt={facility.name}
+                      className="w-full h-full object-cover"
+                    />
+                  ) : (
+                    <div className="w-full h-full flex items-center justify-center overflow-hidden">
+                      <Avatar name={facility.name} />
+                    </div>
+                  )}
+                </div>
+                <CardContent className="p-6">
+                  <div className="flex flex-col h-full">
+                    <div className="flex items-start justify-between">
+                      <div>
+                        <h3 className="text-md font-medium text-gray-900">
+                          {facility.name}
+                        </h3>
+                        <div className="font-medium">
+                          {facility.facility_type}
                         </div>
                       </div>
                     </div>
-                  </CardContent>
-                  <CardFooter className="flex justify-end">
-                    <Link
-                      href={`/facility/${facility.id}`}
-                      className="text-sm text-primary hover:underline"
+                  </div>
+                </CardContent>
+                <CardFooter className="flex justify-between">
+                  <div className="flex items-center text-primary cursor-pointer">
+                    <CareIcon icon="l-edit" className="mr-1 h-4 w-4" />
+                    <p className="hover:underline text-underline-offset-[4px] text-decoration-thickness-[2px]">
+                      <EditFacilitySheet
```

Comment:
Like above, move this out (inside the root for this component) and attach open to click here.